ui/flutter/pubspec.yaml

```diff
@@ -73,10 +75,11 @@ dev_dependencies:
   # rules and activating additional ones.
   flutter_lints: ^2.0.0
   ffigen: ^8.0.2
-  # build json model: dart run build_runner build --delete-conflicting-outputs
+  # build json model: flutter pub run build_runner build --delete-conflicting-outputs
```

Comment:
Flutter pub run has been marked for deprecation, so we used `dart run instead.

Reply:
What should I do to resolve this? 
at the time I have fired up these 2 commands
1. `flutter clean`
2. `dart run build_runner build --delete-conflicting-outputs`

I'm not sure, How do I transform from `flutter pub run` to `dart run`
Could you please through some lights on it?

Reply:
I mean, just keep `dart run`  comment.